src/main/kotlin/no/nav/yrkesskade/ekstern/gateway/ScopeValidator.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.yrkesskade.ekstern.gateway
+
+import no.nav.yrkesskade.ekstern.gateway.config.ScopeValidationConfiguration
+import org.springframework.stereotype.Component
+import org.springframework.util.AntPathMatcher
+import org.springframework.web.server.ServerWebExchange
+
+@Component
+class ScopeValidator(val scopeValidationConfiguration: ScopeValidationConfiguration) {
+
+    fun validateScope(exchange: ServerWebExchange, scopeFromToken: String?): Boolean {
+        val scopesForRoute = scopeValidationConfiguration.scopes[exchange.getRouteId()]
```

Comment:
Hva om det ikke er definert scope/eller behov for scope? Vil du måtte bruke noscope i slike scenarioer?

Reply:
Ja, med dette designet så blir det slik at vi må ha et aktivt forhold til hvilke apier og pather som skal ha hvilke scopes.

Reply:
OK!